modules/installation/pages/hardware-requirements.adoc

```diff
@@ -11,54 +11,141 @@ For {smr} hardware requirements, see xref:retail:retail-requirements.adoc[].
 
 == Server Hardware Requirements
 
-[cols="1,1", options="header"]
+[cols="1,1,1", options="header"]
 .Server Hardware Requirements for {x86_64} Architecture
 |===
-| Hardware                 | Recommended
-| CPU                      | Minimum 4 dedicated 64-bit CPU cores
-| RAM:                     | _Test Server_ Minimum 8{nbsp}GB
-|                          | _Base Installation_ Minimum 16{nbsp}GB
-|                          | _Production Server_ Minimum 32{nbsp}GB
-| Disk Space:              | [path]``/`` _(root)_ Minimum 24{nbsp}GB
-|                          | [path]``/var/lib/pgsql`` Minimum 50{nbsp}GB
-|                          | [path]``/var/spacewalk`` Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
-|                          | [path]``/var/cache`` In accordance to products to be synchronized: 100{nbsp}MB per {suse} product, 1{nbsp}GB per Red Hat or other product.
+
+| Hardware
+| Details
+| Recommendation
+
+| CPU
+| -
+| Minimum 4 dedicated 64-bit CPU cores
+
+| RAM
+| Test Server
+| Minimum 8{nbsp}GB
+
+|
+| Base Installation
+| Minimum 16{nbsp}GB
+
+|
+| Production Server
+| Minimum 32{nbsp}GB
+
+| Disk Space
+| [path]``/`` (root directory)
+| Minimum 24{nbsp}GB
+
+|
+| [path]``/var/lib/pgsql``
+| Minimum 50{nbsp}GB
+
+|
+| [path]``/var/spacewalk``
+| Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
+
+|
+| [path]``/var/cache``
+| Minimum 10{nbsp}GB.
+Add 100{nbsp}MB per {suse} product, 1{nbsp}GB per Red Hat or other product.
 Double the space if the server is an ISS Master.
```

Comment:
If we mention ISS Master here, we must explain what this is (xref?).

Reply:
Oh, that's something I hadn't thought of. I could put an xref below the table? :thinking: